hackerforms/auth.py

```diff
@@ -0,0 +1,17 @@
+from .socket import send, receive
+
+def authenticate():
+  """Get the users email and verify it
+
+  This method will ask the user to enter their email address and will
+  verify it. It will then return the email address and wether the user
+  is authenticated or not.
```

Comment:
```suggestion
  This method will request users to fill in their email addresses and 
  verify them. It will then return the email address and wether the user
  is authenticated or not.
```